internal/pkg/manager/manager.go

```diff
@@ -110,7 +113,10 @@ func (pm *PackManager) ProcessTemplates() (*renderer.Rendered, []*errors.Wrapped
 }
 
 // ProcessOutputTemplate performs the output template rendering.
-func (pm *PackManager) ProcessOutputTemplate() (string, error) { return pm.renderer.RenderOutput() }
+func (pm *PackManager) ProcessOutputTemplate(aux bool) (string, error) {
+	pm.renderer.RenderAuxFiles = aux
```

Comment:
```suggestion
func (pm *PackManager) ProcessOutputTemplate() (string, error) {
```

I'm still getting used to the code base and the lifecylce of each component, but it seems like `ProcessOutputTemplate` is expected to only be called after `ProcessTemplates` (otherwise `pm.renderer` is `nil`), and so `RenderAuxFiles` should also be expected to be set?

Reply:
You are right of course! Fixed in c273f650df96ecd57b64bfe0486846badebb4466